### PR TITLE
[WIP] Mdsa training

### LINKED ISSUE #85: Draft approach and plan for training of MDSA participants
This is an issue to document draft thinking and planning for how we'll deploy training to participants in MDSA

https://docs.google.com/document/d/1XiHYz0BCvdslOdW9WrF7fO2XuPIHNYEHp7NfLqzo9jQ/edit#

### PR DESCRIPTION
**Pull request by ian-r-rose**
This is still very much a work in progress, but I wanted to move the discussion around it forward. I'm not really happy with the dbt fundamentals portion, I feel like we'd still want to have people on a call to make sure that they can connect to the project using their credentials.

Fixes #85, supersedes #78

### CODE REVIEW

File: docs/learning/training.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+* Become familiar with Snowflake terminology (warehouse, database, role etc).
+* Have a basic understanding of how to use the Snowflake user interface.
+
+**Who should go through this?**
+Data engineers should go through this training,
+
+### 1. Take Snowflake's [Data Warehousing Workshop](https://learn.snowflake.com/en/courses/uni-essdww101/)
+
+This covers the fundamentals of the Snowflake Data Warehouse.
+It should take 6-8 hours.
```
Comment (ian-r-rose) [inline docs/learning/training.md:73]: @britt-allen I don't have much information about the content here, do you have anything you want to add about this workshop?

Comment (britt-allen) [reply docs/learning/training.md:73]: Yes, I can copy content from here: https://docs.google.com/document/d/1fCS0_8WANH1YZF53Jk006e0UHoygnb_sJvUF4olF-jc/edit

**Comment by ian-r-rose** [comment]
Closing as this has been superseded by a slide deck for the time-being

**Comment by britt-allen** [comment]
Hey @ian-r-rose, can I get the link to the slide deck, please?

**Pull request closed without merging**